src/main/java/picard/vcf/MergeVcfs.java

```diff
@@ -94,6 +94,8 @@ public MergeVcfs() {
     protected int doWork() {
         final ProgressLogger progress = new ProgressLogger(log, 10000);
         final List<String> sampleList = new ArrayList<String>();
+        INPUT = IOUtil.unrollFiles(INPUT, IOUtil.VCF_EXTENSIONS);
```

Comment:
Came here to say this.  Could you just add a mention of the capability to the doc for the option?